invenio_matcher/core.py

```diff
@@ -24,78 +24,57 @@
 
 """Matcher core."""
 
-from invenio.ext.es import es
+import copy
 
-from .errors import InvalidQuery
-from .utils import build_exact_query, build_fuzzy_query
+from invenio_matcher.engine import exact
```

Comment:
RE imports: yes, I tried to do something weird with the imports to see what they looked like. Turns out, pretty bad. Will revert to something more sane.